app.js

```diff
@@ -1 +1,10 @@
-function calculateFactorial(number) {}
+function calculateFactorial(number) {
+  let fact = 1;
+  for (let i = 2; i <= number; i++) {
```

Comment:
i should start from 1, not 2, as factorial calculation begins from 1 and includes all integers up to the factorial number.